### PR TITLE
adding windsor connector

### PR DESCRIPTION
**Pull request by mkzsg**

### CODE REVIEW

**Comment by Court72** [comment]
@mkzsg - Thank you for your contribution. Would you take a moment to sign the [Contributor License Agreement (CLA)](https://github.com/MicrosoftDocs/powerquery-docs/pull/552#issuecomment-2180321883)? After the CLA is signed, someone can review your pull request. Thanks!

**Comment by Court72** [comment]
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by mkzsg** [comment]
@microsoft-github-policy-service agree [company="Windsor.ai"]

**Comment by mkzsg** [comment]
@microsoft-github-policy-service agree company="Windsor.ai"

**Comment by JasonWHowell** [comment]
Close open to trigger a build.

File: powerquery-docs/connectors/windsor.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+5. When you are satisfied with the result, click on Save Query button, give your query a recognizable name and click "Save".
+    
+    :::image type="content" source="media/windsor/windsor-save-query.png" alt-text="Save Query at Windsor Onboard.":::
+
+    The queries you save here are available to you in Power BI Desktop once you sign in to the Windsor connector.
+
+
+## Capabilities supported
+* Import 
+* Data import from multiple sources
```
Comment (ptyx507x) [inline powerquery-docs/connectors/windsor.md:50]: You can delete this line. You can either have Import mode and/or DIrectQuery. "Data import from multiple sources" would fall in the "Import" category

**Review by ptyx507x — CHANGES_REQUESTED** [review]

**Comment by ptyx507x** [comment]
quick ping @mkzsg to please address the comments shared

**Comment by mkzsg** [comment]
> quick ping @mkzsg to please address the comments shared

@ptyx507x pushed fixes. hope this is ready to go now.

**Comment by DougKlopfenstein** [comment]
@mkzsg - You also need to add the Windsor connector to the list of all connectors in the Power Query documentation. The list of all connectors can be found in the connectors/index.md file. The list is in alphabetical order. Also, since your Windsor icon (windsor-64.png) is black with a transparent background, be aware that if someone is using dark mode in their web browser, your icon will disappear against the black background. If you want it to remain visible in dark mode, you should probably use a white background.

**Comment by DougKlopfenstein** [comment]
@mkzsg - Also, since this is a beta release of your connector, you should add "(Beta)" to the Windsor name in the table of contents (toc.yml), the list of all connectors (connectors/index.md), and to heading 1 of the connector article.

**Comment by mkzsg** [comment]
@DougKlopfenstein fixed.

**Comment by mkzsg** [comment]
@DougKlopfenstein can this be merged now?

**Comment by DougKlopfenstein** [comment]
@mkzsg 
I've done an edit pass to check spelling, grammar, formatting, and Microsoft style. Please take a look at the article to make sure that I didn't change the meaning of some of the more esoteric sentences. If they are OK, just leave a comment here and I'll merge the PR.

**Comment by mkzsg** [comment]
> @mkzsg I've done an edit pass to check spelling, grammar, formatting, and Microsoft style. Please take a look at the article to make sure that I didn't change the meaning of some of the more esoteric sentences. If they are OK, just leave a comment here and I'll merge the PR.

Yes, all OK. LGTM. Let's merge.

**Review by DougKlopfenstein — APPROVED** [review]

**Comment by JasonWHowell** [comment]
@Court72 can you review sometime today? Thanks, Jason

**Comment by Jak-MS** [comment]
@DougKlopfenstein @JasonWHowell - usually PR review would prefer to move the commits for new articles to the private repo. Getting a staged page ensures that (among other things) images display correctly and Acrolinx helps us check for editorial issues like unapproved acronyms, typos, and spelling errors. That said, we'll merge this with the following recommendation:

@mkzsg - can you check the article after it goes live to ensure all the images display correctly and the page renders as you intended? thanks!

**Pull request merged**